contracts/standard/arbitration/MultipleArbitrableTokenTransaction.sol

```diff
@@ -66,7 +66,7 @@ contract MultipleArbitrableTokenTransaction is IArbitrable {
      *  @param _amount The amount paid.
      *  @param _party The party that paid.
      */
-    event Payment(uint _transactionID, uint _amount, address _party);
+    event Payment(uint indexed _transactionID, uint _amount, address indexed _party);
```

Comment:
We don't need to index the party, there are only 2 so it would be a waste of gas.